connectors/src/connectors/google_drive/temporal/file.ts

```diff
@@ -46,6 +46,11 @@ export async function syncOneFile(
     },
   });
 
+  // Early return if lastSeenTs is greater than activity start, indicating probable failure.
```

Comment:
```suggestion
  // Early return if lastSeenTs is greater than workflow start.
  // This allows avoiding resyncing already-synced documents in case of activity failure
```
nit: clearer com

Reply:
If we fail processing a spreadsheet but still mark it in the database with a new lastSeenTs, isn't that another bug?

Reply:
The spreadsheet isn't updated in DB until it is successfully synced and upserted.

Reply:
So how is that going to work? If it fails it won't update the lastSeen. Did I miss someething?